### PR TITLE
To do list: Add remove

### PR DESCRIPTION
**Pull request by algerina**
In this Branch I implemented the following:
-  Removed hardcoded items from the tasks array.
- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task.
- Implemented a function for editing task descriptions.
- Implemented a function for deleting a task (remove an element from the array).
- Implemented a function for the "Clear all completed" button.
- Deleting a task update all remaining items' indexes, so they represent the current list order and are unique.
- All changes to the To-Do List are be saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -34,17 +34,55 @@ function createTask() {
...
+
+// function for delete all completed
+function deleteCompleted() {
+  let tasks = JSON.parse(localStorage.getItem('tasks'));
+  const newList = tasks.filter((task) => task.completed === false);
+
+  updatePosition(newList);
+  renderTasks(newList);
+  saveList(newList);
+}
```
Comment (safafa) [inline src/index.js:84]: - [x] Well done on implementing the functionalities. However, as mentioned in the project's requirements you have to create a new file for the new functionality. Using ES6 modules will help make the code clean and easy to maintain. Kindly reorganize your code.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @algerina , You did a great job so far 👏🏼 👏🏼 

- Well done the app is working as expected

However, here's one point you need to work on to get your project approved.

## Required changes :

Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (algerina) [reply src/index.js:84]: can explain more please because I think that I already did it in render and storage files

Comment (safafa) [reply src/index.js:84]: Sorry my bad  you can request for a new review

Comment (algerina) [reply src/index.js:84]: Thank you

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @algerina, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**